pkgs/development/python-modules/mysqlclient/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, libmysql }:
+
+buildPythonPackage rec {
+  pname = "mysqlclient";
+  version = "1.3.12";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.